# First time it prints 3. Second time is an error because x is locally scoped in the block and can't be accessed outside. 